.github/workflows/ui_tests.yml

```diff
@@ -110,9 +86,86 @@ jobs:
         env:
           EIP155_PRIVATE_KEY: ${{ secrets.EIP155_PRIVATE_KEY }}
           SOLANA_PRIVATE_KEY: ${{ secrets.SOLANA_PRIVATE_KEY }}
+          NODE_OPTIONS: '--max-old-space-size=4096' # Increase memory for webpack
```

Comment:
any drawback on this?

Reply:
I needed this to build locally via `act` - don't think there's a drawback